data/human/human missions.txt

```diff
@@ -4175,3 +4175,166 @@ mission "Hai Wormhole Warning"
+				goto choice
+			label find
+			`	"Lucky for you, Lokust does not seem particularly attached to his 'booty'. If you must, offer him a large sum of money, and he will most likely be willing to part with them. However, if there is something you can do to get those statues back without allowing your money to fall into the hands of a criminal, I would suggest that course of action instead."`
+				goto choice
+			label accept
+			`	"Excellent. You're doing a service to humanity's cultural memory."`
+				accept
+	on complete
+		conversation
+			`You hustle through the crowded markets, pistol by your side, looking for any sign of the statues. After a while you come across a storefront with a crudely painted sign that reads "Fine Art and Cultural Treasures". Not seeing any better leads, you go in.`
+			`	The inside is damp and poorly lit, but you can make out various paintings, manuscripts, and other expensive-looking items. "How can I help you?" the store clerk asks in a disarmingly friendly voice."`
+			choice
+				`	"You don't happen to know of a local warlord named Lokust, do you?"`
+				`	"Have you had any Greek statues pass by through here?"`
+			`	The clerk laughs. "You're trying to get Alexandria's shipment back, aren't you?" Before you can answer, he continues. "If you want my client Lokust's statues, I'm afraid you're gonna have to pay a pretty penny for them. 500,000 credits is his asking price."`
+			choice
+				`	"Alright, I'll pay."`
+					goto pay
+				`	"Can't we negotiate?"`
+				`	"I can't pay that much!"`
+			`	"Sorry, his price is firm. I'm just the dealer. He's the one pulling the strings." A smirk begins to crawl across his face, and he leans in. "Look, I have no love for Lokust. Always jacking up his cut of the sale, threatening my customers, you get the idea. Not to mention my debts to him." The last part he says in a less bold tone. "What I'm saying is, is that I wouldn't mind if someone like you took him out. He's a real wacko, always going on about duels and other bull. If I were you, I'd challenge him to one. If he doesn't accept, call him a coward. That's sure to get him riled up." He wheezes in a way vaugely reminiscent of a laugh. "As a reward, I'll give you the statues for free if you come back, but you gotta take out his whole crew, right? Don't want any of his underlings coming for me."`
+			choice
+				`	"Never mind, I'll pay."`
+					goto pay
+				`	"All right, I'll fight him. `
+			`	"Heh, good choice. If you're worried about his crew, don't. I'm sure if Alexandria thinks you're good enough to hire, you can handle it. I'll call him up so you don't have to go chasing him down." He pulls up a hologram communicator and noticeably stiffens up. "Hello Lokust. Hope you're not too busy right now, because someone wants to see you." He turns the communicator towards you, and you see a man in clothes more befitting of a prince than a pirate.`
```

Comment:
```suggestion
			`	"Heh, good choice. If you're worried about his crew, don't be. I'm sure if Alexandria thinks you're good enough to hire, you can handle it. I'll call him up so you don't have to go chasing him down." He pulls up a hologram communicator and noticeably stiffens up. "Hello Lokust. Hope you're not too busy right now, because someone wants to see you." He turns the communicator towards you, and you see a man in clothes more befitting of a prince than a pirate.`
```